.gitlab-ci.yml

```diff
@@ -33,85 +33,31 @@ build:
     DOCKER_DRIVER: overlay2
```

Comment:
this stage is still called `build` even though it is doing a lot more now. also, the file announces three stages (`build`, `test`, `website`) but only does one now. can you please clean this up before we merge it?